ssh_automated_tests/core/user.py

```diff
@@ -16,3 +20,13 @@ def make_connection_data(self):
         return {"hostname": self._address, "port": self._port,
                 "username": self._name, "key": self._key}
 
+
+class SshUserParser(object):
```

Comment:
@extsoft Well, seems I got confused around this implementation.
I see that it's possible to avoid creating extra classes like `SshUserParser`.
What do you think? Should I remove that class or use it instead?